CHANGELOG.md

```diff
@@ -59,12 +59,12 @@ Bugfixes
 ## 2.4.0 (31/07/2017)
 
 Features
-  - Add functionality to track affected user in Sidekiq jobs, refer to the README for more information, under the "Affected User Tracking in Sidekiq" heading ([#121](https://github.com/MindscapeHQ/raygun4ruby/pull/121))
+  - Add functionality to track affected user/customer in Sidekiq jobs, refer to the README for more information, under the "Affected User Tracking/Customers in Sidekiq" heading ([#121](https://github.com/MindscapeHQ/raygun4ruby/pull/121))
```

Comment:
Wondering why the changes in this file use the slash instead of replacing the "affected user" text?

Reply:
As this is a changelog I thought it would be best to not remove content from here. Instead add more to

Reply:
Cool, makes sense to me 😃